conf/janus.eventhandler.nngevh.jcfg.sample

```diff
@@ -1,9 +1,9 @@
-# This configures the Nanomsg event handler. Since this plugin only
-# forwards each event it receives via Nanomsg, you simply need to
+# This configures the NNG event handler. Since this plugin only
+# forwards each event it receives via NNG, you simply need to
 # configure (i) which events to subscribe to, (ii) the address to use for
 # the communication, and (iii) whether the address should be used to bind
 # locally or to connect to a remote endpoint. Notice that the only supported
-# pattern is NN_PUBSUB, where the Nanomsg event handler is the publisher.
+# pattern is NN_PUBSUB, where the NNG event handler is the publisher.
```

Comment:
This should probably say NNG_PAIR, https://github.com/meetecho/janus-gateway/pull/2242/files#diff-5b84e446e3e9cf404a19e5947bef4edfR205